css/templates/lune/pages/_landing.css

```diff
@@ -30,12 +30,37 @@
  * Layout & typography
  ******************************************************************************/
 .lune-page-landing {
+  height: 100vh;
```

Comment:
Adding a wrapper to prevent the <body> from scrolling. Fixes a bug reported by QA about the content on the page jumping when scrolling on iOS / Android Chrome. The bug happens because those browsers collapse their URL bars when the body scrolls. Adding this wrapper fixes the bug